usr/bin/irdata_text_index.py

```diff
@@ -74,18 +62,9 @@
         f_in = sys.stdin
 
     # Get the field to be treated as the indexed term.
-
-    if "index" in args:
-        field = index_for_int(args["index"])
-    else:
-        field = 0
-
+    field = data.index_for_int(args.f) if args.f is not None else 0
     # Get the field separator.
-
-    if "separator" in args:
-        separator = args["separator"]
-    else:
-        separator = "\t"
+    separator = args.d  # separator
```

Comment:
Adding the `dest` attribute to `add_argument()` earlier avoids the need for this comment.
```suggestion
    separator = args.separator
```